src/repositories/seo-links-repository.php

```diff
@@ -152,6 +152,10 @@ public function get_incoming_link_counts_for_post_ids( $post_ids ) {
 	 * @return array An array of associative arrays, each containing a indexable id and incoming property.
 	 */
 	public function get_incoming_link_counts_for_indexable_ids( $indexable_ids ) {
+		if ( empty( $indexable_ids ) ) {
```

Comment:
Maybe we need to check also ! \is_array( $indexable_ids ) here? And add test case for non array input?

Reply:
Should that be a check for an array or for something which is array accessible or iterable ?

☝🏻 This was the reason for me not to go down that rabbit hole at this time. For now, I've solved the immediate problem. Further improvements, accompanied by tests proving the need for those improvements, are definitely welcome ;-)